### ISSUE TITLE
Properties of custom content

### ISSUE THREAD

**Issue by salarenko** [teodosii/react-notifications-component#138]
Hello, I could't find the answer in the documentation.

I'd like to use custom component for displaying the notification. In my custom component I'd like to have access to `title`, `message`, `type` and general configuration of the component, passed to the `addNotification`.

My first thought was, the setup is going to be passed as arguments to the custom component, but after a little POC I see two argument being passed: `{ id: someId }` and empty object `{}`.

Is it expected behaviour?
What is the easiest way to access configuration of the notification from the custom component?

Little POC:
```
const CustomContentComponent = (...args) => {
  console.log(args);    // => [{id: '10'}, {}]
  return <div/>
}
```

```
store.addNotification({
      title: 'Some title',
      type: 'success',
      message: 'Some message',
      content: CustomContent,
      container: 'top-right',
      animationIn: ['animate__animated', 'animate__fadeIn'],
      animationOut: ['animate__animated', 'animate__fadeOut'],
      dismiss: { duration: 5000 }
    })
```

**Comment by AndreaEsposit** [comment]
How do we pass the notification-ID and custom props to content?

**Comment by teodosii** [comment]
Fixed in `3.4.0`, use `notificationConfig` from props.
